src/modules/material-requests/dto/material-requests.dto.ts

```diff
@@ -19,14 +27,15 @@ export class MaterialRequestsQueryDto {
 	query?: string;
 
 	@IsString()
-	@Type(() => String)
+	@IsEnum(MaterialRequestType)
 	@IsOptional()
 	@ApiPropertyOptional({
-		type: MaterialRequestType,
+		type: String,
 		description: 'Which type of material request is requested',
 		default: MaterialRequestType.VIEW,
+		enum: MaterialRequestType,
 	})
-	type?: string;
+	type? = undefined;
```

Comment:
the default in the ApiPropertyOptional is set to default: MaterialRequestType.VIEW, but the default in code is set to undefined.

Best to take undefined, so we can omit the filter and get all results back independant of type